models/public/openpose-cmu-tf/openpose-cmu-tf.md

```diff
@@ -0,0 +1,80 @@
+# openpose-cmu-tf
+
+## Use Case and High-Level Description
+
+This is TensoFlow version of OpenPose algorithm, used for multi-person
+human pose estimation. The model is based on VGG pretrained network,
+which described in the original [paper](https://arxiv.org/pdf/1812.08008.pdf).
+For details see [repository](https://github.com/ildoonet/tf-pose-estimation).
+## Example
+
+## Specification
+
+| Metric            | Value                 |
+|-------------------|-----------------------|
+| Type              | Human pose estimation |
+| GFLOPs            | 319.304               |
+| MParams           | 52.311                |
+| Source framework  | TensorFlow\*          |
+
+## Accuracy
+
+## Performance
+
+## Input
+
+### Original mode
+
+Image, name - `image`,  shape - `1,368,432,3`, format is `B,H,W,C` where:
+
+- `B` - batch size
+- `H` - height
+- `W` - width
+- `C` - channel
+
+Channel order is `BGR`.
+
+### Converted model
+
+Image, name - `image`,  shape - `1,3,368,432`, format is `B,C,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `BGR`.
+
+## Output
+
+### Original model
+
+Concatenation of keypoint pairwise relations (part affinity fields) and keypoint heatmaps,
+name - `Openpose/concat_stage7`, output data format is `B,H,W,C` where:
+
+- `B` - batch size
+- `H` - height
+- `W` - width
+- `C` - heatmap channel
+
+### Converted model
+
+1. Predicted keypoint heatmaps, name - `Mconv7_stage6_L2/Conv2D`, shape - `1,19,46,54`,
+   output data format is `B,C,H,W` where:
+
+- `B` - batch size
+- `C` - number of predicted body parts according to MSCOCO kepoints classification
```

Comment:
it is not correct:
* `MSCOCO keypoints` challenge works only with 17 keypoints (no keypoint for neck, the last for backgroud):
0 | "nose"
1 | "left_eye"
2 | "right_eye"
3 | "left_ear"
4 | "right_ear"
5 | "left_shoulder"
6 | "right_shoulder"
7 | "left_elbow"
8 | "right_elbow"
9 | "left_wrist"
10 | "right_wrist"
11 | "left_hip"
12 | "right_hip"
13 | "left_knee"
14 | "right_knee"
15 | "left_ankle"
16 | "right_ankle"
* the keypoints order is not equvivalent to COCO
see: https://github.com/CMU-Perceptual-Computing-Lab/openpose/blob/master/doc/output.md#pose-output-format-coco

Reply:
for better understanding, [mapping keypoints to coco order in AC](https://github.com/opencv/open_model_zoo/blob/master/tools/accuracy_checker/accuracy_checker/annotation_converters/cvat_human_pose.py#L23-L41)